Code/GraphMol/MMPA/MMPA.cpp

```diff
@@ -329,7 +333,17 @@ static inline void processCuts(
 
 bool fragmentMol(const ROMol& mol,
                  std::vector<std::pair<ROMOL_SPTR, ROMOL_SPTR> >& res,
-                 unsigned int maxCuts, unsigned int maxCutBonds,
+                 unsigned int maxCuts,
+                 unsigned int maxCutBonds,
+                 const std::string& pattern) {
+  return fragmentMol(mol, res, 1, maxCuts, maxCutBonds, pattern);
+}
+
+bool fragmentMol(const ROMol& mol,
+                 std::vector<std::pair<ROMOL_SPTR, ROMOL_SPTR> >& res,
+                 unsigned int minCuts,
+                 unsigned int maxCuts,
+                 unsigned int maxCutBonds,
                  const std::string& pattern) {
```

Comment:
seems like you should either have a `PRECONDITION(minCuts<=maxCuts,"min not <= max") on this function or a check that throws a ValueError if that condition isn't true. Maybe the second is better.
Either way including a test for it isn't a bad idea.